app/serializers/optimized_project_serializer.rb

```diff
@@ -25,10 +30,13 @@ class OptimizedProjectSerializer
     status
   ].freeze
 
-  def initialize(projects, requested_name_map, internal_key: false)
+  VERSION_ATTRIBUTES = %i[published_at number original_license status].freeze
```

Comment:
alternatively, we could let the caller pass in the attributes they want? Might avoid API awkwardness in the future 🤔

Reply:
or include these as baseline and anything beyond these can be asked for?